api/openapi/jobs.yaml

```
@@ -83,12 +83,44 @@ paths:
             $ref: "#/components/schemas/Id"
           required: true
       responses:
-        "200":
+        200:
           description: A JSON object representing an Ensembling Job
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/EnsemblingJob"
+        404:
+          description: "Invalid ensembling job"
+
+    delete:
+      tags: *tags
+      operationId: "DeleteEnsemblingJob"
+      summary: Deletee an ongoing Ensembling Job.
+      parameters:
+        - in: path
+          name: project_id
+          schema:
+            $ref: "#/components/schemas/Id"
+          required: true
+        - in: path
+          name: id
```

```suggestion
          name: job_id
```
It's better to avoid ambiguity by explicitly specifying id of what is expected here

---

ok